lib/helpers/backend/makeAxiosRequest.ts

```diff
@@ -0,0 +1,34 @@
+import { AxiosRequestConfig } from "axios";
+import { StatusCodes } from "http-status-codes";
+import axios from "axios";
+
+export async function makeAxiosRequest(
+  endpoint: string,
+  options?: AxiosRequestConfig,
+  milliInterval?: number,
+  maxRetry?: number
+) {
+  maxRetry = maxRetry || 5;
+  milliInterval = milliInterval || 1000;
+  options = options || {};
+  async function getData(maxRetry: number) {
+    return await axios
```

Comment:
Since this function is asynchronous, you can `return` the `Promise<Data | null>` directly without needing to `await` it.

Reply:
File removed. Replaced with googleapi library.